dispatcher/frontend-ui/src/constants.js

```diff
@@ -15,8 +16,22 @@ function format_duration(diff) { // display a duration in a standard format
 
 function format_duration_between(start, end) { // display a duration between two datetimes
   let diff = moment(end).diff(start);
-  return format_duration(diff);
-}
+   const humanize_duration = humanizeDuration.humanizer({
+      language: "shortEn",
+      languages: {
+        shortEn: {
+          y: () => "y",
+          mo: () => "mo",
+          w: () => "w",
+          d: () => "d",
+          h: () => "h",
+          m: () => "m",
+          s: () => "s",
+          ms: () => "ms",
+        },
+      },largest: 2 ,round : true ,delimiter: " "});
```

Comment:
Please use

``` js
}, largest: 2, round: true , delimiter: " ", spacer: ""});
```
so we keep our code formatting and adjust the displayed string as well.